src/lightbox/tp-lightbox.ts

```diff
@@ -199,6 +202,9 @@ export class TPLightboxElement extends HTMLElement {
 		// Now, show the modal.
 		dialog.showModal();
 		this.setAttribute( 'open', 'yes' );
+
+		// Update navigation current item.
+		this.updateNavCurrentItem();
```

Comment:
@nirav7707 Let's listen to the attribute instead of calling this multiple times

Reply:
Update Done!